OpenShift-Installation/eVDF-Openshift-Installation.rst

```diff
@@ -0,0 +1,592 @@
+
+=============================================
+eVDF OpenShift Installation - Standalone and HA
+=============================================
+
+.. contents::
+   :local:
+   :depth: 3
+   
+
+Supported Platforms
+====================
+
+The VSP integration with eVDF on OpenShift is supported on RHEL Server hosts (Version 7.4).
```

Comment:
I'm missing details on how to lock the hosts on 7.4 (or how to handle the docker and selinux issues?), if these are in the other OpenShift guide, please reference to them here